### PR TITLE
[PLAT-5429] Add GTM-tracking

### PR DESCRIPTION
**Pull request by osa911**
[PLAT-5429](https://reevoo.atlassian.net/browse/PLAT-5429)

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -6,3 +6,6 @@ CP_ANALYTICS_API=http://localhost:15800/api/v1/
 # Tableau
 TABLEAU_GATEWAY_API=http://localhost:9595/api/v1/
 TABLEAU_HOST=https://tableau-testing.reevoo.com/
+GTM_ENV=env-23
+GTM_AUTH_KEY=jzL1Clp8e0DesAtc9Ns_kQ
```
Comment (mPanasiewicz) [inline .env.development:10]: @osa911 Is this fake Auth Key??

File: .env.production [hunk 2]
```diff
@@ -6,3 +6,6 @@ CP_ANALYTICS_API=https://cp-analytics.reevoo.com/api/v1/
 # Tableau
 TABLEAU_GATEWAY_API=https://tableauapigateway.reevoo.com/api/v1/
 TABLEAU_HOST=https://tableau.reevoo.com/
+GTM_ENV=env-1
+GTM_AUTH_KEY=CmO5CrJ2SMIyrU8Ko9t6ag
```
Comment (mPanasiewicz) [inline .env.production:10]: @osa911 I think production AuthKey shouldn't be stored in the repository

File: .env.staging [hunk 3]
```diff
@@ -6,3 +6,6 @@ CP_ANALYTICS_API=https://cp-analytics-staging.reevoocloud.com/api/v1/
 # Tableau
 TABLEAU_GATEWAY_API=https://tableauapigateway-staging.reevoocloud.com/api/v1/
 TABLEAU_HOST=https://tableau-testing.reevoo.com/
+GTM_ENV=env-21
+GTM_AUTH_KEY=HkGbzImqRSXBnzwbiTCqaw
```
Comment (mPanasiewicz) [inline .env.staging:10]: @osa911 Is this fake or test Auth Key??

File: .env.development (hunk 1 above)
Comment (osa911) [reply .env.development:10]: It's real auth key for development env in GTM

File: .env.staging (hunk 3 above)
Comment (osa911) [reply .env.staging:10]: It's real auth key for stage1 env in GTM

File: .env.production (hunk 2 above)
Comment (osa911) [reply .env.production:10]: Maybe you are right, but repo is private
And I think it's ok.
This is necessary mostly for debugging tracking and that's all
Main key is GTM_ID

**Review by mPanasiewicz — APPROVED** [review]
Looks Good To ME

**Pull request merged**